bin/concat_prefactor_files

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+"""
+Script to concatenate the MS files from Prefactor for input to Rapthor
+"""
+import argparse
+from argparse import RawTextHelpFormatter
+import glob
+import os
+import sys
+import subprocess
+import casacore.tables as pt
+import numpy as np
+
+
+def main(input_path, output_file):
+    """
+    Concatenate MS files from Prefactor for input to Rapthor
+
+    Parameters
+    ----------
+    input_path : str
+        Full path to the directory with the Prefactor MS files
+    output_file : str
+        Filename of output file
+    """
+    msfiles = glob.glob(os.path.join(input_path, '*.ms'))
+
+    # Order the files by frequency, filling any gaps with dummy files
+    first = True
+    nchans = 0
+    freqs = []
+    for ms in msfiles:
+        # Get the frequency info
+        sw = pt.table(ms+'::SPECTRAL_WINDOW', ack=False)
+        freq = sw.col('REF_FREQUENCY')[0]
+        if first:
+            file_bandwidth = sw.col('TOTAL_BANDWIDTH')[0]
+            nchans = sw.col('CHAN_WIDTH')[0].shape[0]
+            chwidth = sw.col('CHAN_WIDTH')[0][0]
+            first = False
+        else:
+            assert file_bandwidth == sw.col('TOTAL_BANDWIDTH')[0]
+            assert nchans == sw.col('CHAN_WIDTH')[0].shape[0]
+            assert chwidth == sw.col('CHAN_WIDTH')[0][0]
+        freqs.append(freq)
+        sw.close()
```

Comment:
Not sure how important this is, but the table will not be closed when an exception is raised; e.g. when an `assert` is triggered.

Reply:
Well spotted -- I will move the close() before the checks to be sure.